ecommerce/pathways/templates/oscar/checkout/cancel_checkout.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'edx/base.html' %}
+{% load core_extras %}
+{% load i18n %}
+{% load django_markup %}
+{% load static %}
+{% settings_value 'payment_support_url' as support_url %}
+
+{% block title %}
+    {% trans "Checkout Cancelled" as tmsg %}{{ tmsg | force_escape }}
+{% endblock title %}
+
+{% block navbar %}
+    {% include 'edx/partials/_student_navbar.html' %}
+{% endblock navbar %}
+
+{% block content %}
+<div class="container content-wrapper receipt-cancel-error">
+    <h1>{% trans "Checkout Cancelled" as tmsg %}{{ tmsg | force_escape }}</h1>
+    <p>
+        {% blocktrans asvar tmsg %}
+        Your transaction has been cancelled. If you feel an error has occurred, go to {start_link}
+        {support_url}{end_link}.
+        {% endblocktrans %}
+        {% interpolate_html tmsg start_link="<a class='nav-link' href='"|add:support_url|add:"'>"|safe end_link="</a>"|safe support_url=support_url|safe %}
+    </p>
+</div>
+{% endblock content %}
```

Comment:
Missing newline at the end of the file. @Jacatove

Reply:
@Squirrel18 Done.